views/templates/hook/post-comment-modal.tpl

```diff
@@ -87,31 +87,21 @@
           </div>
 
           <div class="row">
-            {if !$logged}
-              <div class="col-md-8 col-sm-8">
-                <label class="form-label" for="comment_title">{l s='Title' d='Modules.Productcomments.Shop'}<sup class="required">*</sup></label>
-              </div>
-              <div class="col-md-4 col-sm-4">
-                <label class="form-label" for="customer_name">{l s='Your name' d='Modules.Productcomments.Shop'}<sup class="required">*</sup></label>
-              </div>
-            {else}
-              <div class="col-md-12 col-sm-12">
-                <label class="form-label" for="comment_title">{l s='Title' d='Modules.Productcomments.Shop'}<sup class="required">*</sup></label>
-              </div>
-            {/if}
+            <div class="col-md-12 col-sm-12">
```

Comment:
The problem with this fix is that it changes the desktop behavior.
We must respect the original design which shows both fields on the same line.

Maybe you could try using bootstrap columns, something like `<div class="col-md-8 col-sm-12">`
So that on mobile each field has its own "row", but the design remains unchanged on large screens

What do you think?

Reply:
Please don't mind because I think,

> We must respect the original design which shows both fields on the same line.

With your suggested bootstrap columns we can't maintain original design which shows both fields on the same line.
It will shows each field has its own "row" but with less width like as screenshot below.
![1](https://user-images.githubusercontent.com/40850057/60439640-bfb23000-9c30-11e9-80ce-78ac6d155346.png)


But if we try using bootstrap columns, something like `<div class="col-md-8 col-sm-6 col-xs-6">` for Title and `<div class="col-md-4 col-sm-6 col-xs-6">` for customer name then both fields will be on the same line like as screenshot below.
![2](https://user-images.githubusercontent.com/40850057/60439957-60a0eb00-9c31-11e9-9cd4-1e2940365ba2.png)

Now, what do you think which is good?

Reply:
This might do it!
Let's confirm with our designer @TristanLDD, what do you think?

Reply:
> This might do it!
> Let's confirm with our designer @TristanLDD, what do you think?

Okay :)

Reply:
Hi, sorry for responding late
The best practice for input fields on mobile is one input on one raw, so if we have to choose it would make more sense to make the mobile modal that way and to change the desktop behavior.

Reply:
@jolelievre what do you think on @TristanLDD review?